assets/css/twenty-twenty.scss

```diff
@@ -2061,6 +2061,10 @@ a.reset_variations {
 	section {
 		padding-top: 2rem;
 		padding-bottom: 0;
+
+		&.up-sells.upsells.products {
```

Comment:
It doesn't fix the problem for all sections in a product page, since almost the same happens with related products too.
Probably better change to something like

```scss
.woocommerce {
	&.single-product section {
		float: left;
	}
...
```

Reply:
Note that using `float:left` only for the upsells will make related products go up:

![Screenshot from 2020-05-04 23-18-02](https://user-images.githubusercontent.com/1264099/81030000-89030280-8e5d-11ea-9de6-aec2c6093788.png)

Reply:
Yeah, it will be better.